flang/include/flang/Common/enum-class.h

```diff
@@ -63,9 +63,8 @@ constexpr std::array<std::string_view, ITEMS> EnumNames(const char *p) {
   [[maybe_unused]] static constexpr std::size_t NAME##_enumSize{ \
       ::Fortran::common::CountEnumNames(#__VA_ARGS__)}; \
   [[maybe_unused]] static inline std::string_view EnumToString(NAME e) { \
-    static const constexpr char vaArgs[]{#__VA_ARGS__}; \
     static const constexpr auto names{ \
-        ::Fortran::common::EnumNames<NAME##_enumSize>(vaArgs)}; \
```

Comment:
Was there any reason this was written this way in the first place? Did you find the original review that introduced this?
(just want to make sure this wasn't initially split to work around another issue).

Reply:
https://reviews.llvm.org/D137859